src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -11030,10 +11334,9 @@ public static Vector512<T> Invoke(Vector512<T> x)
             //        e^x = (2^N) * (2^r)
 
             private const uint V_ARG_MAX = 0x42AE0000;
-            private const uint V_MASK = 0x7FFFFFFF;
 
             private const float V_EXPF_MIN = -103.97208f;
-            private const float V_EXPF_MAX = 88.72284f;
+            private const float V_EXPF_MAX = +88.72284f;
```

Comment:
This is just style, right?

Reply:
Correct, just better aligning the numbers, which I personally find more visually appealing/easier to read.